python-3.12.yaml

```diff
@@ -48,7 +48,7 @@ var-transforms:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 8e8a4baf652f6e1cee7acde9d78c4b6154539748
+      expected-commit: ff3bc82f7c9882c27aad597aac79355da7257186
       repository: https://github.com/python/cpython.git
```

Comment:
@smoser Why no --with-wheel-pkg-dir=/usr/share/python-wheels here in python-3.12?

Reply:
> Things to note:
>
> *  python3.12 no longer has setuptools included https://github.com/python/cpython/issues/95299.
> *  python3.12 is at least currently updating the bundled pip wheel.

I'm happy to do that if you think we should. and just not add the ensurepip-wheels dep?

Reply:
my head hurts, one of those ways is deprecated right?!

Reply:
No. It's not related to depreciation as setup tools is still a build backend. For now not doing 3.12 is what I suggested I wouldn't block getting the older versions fixed here